webrecorder/webrecorder/downloadcontroller.py

```diff
@@ -246,16 +246,13 @@ def wasapi_list(self):
         download_path = self.get_origin() + '/api/v1/download/{user}/{coll}/{filename}'
 
         for collection in colls:
-            if commit:
-                commit_id = collection.commit_all()
-                while commit_id:
-                    gevent.sleep(10)
-                    commit_id = collection.commit_all(commit_id)
+            commit_storage = collection.get_storage()
+            local_storage = LocalFileStorage(self.redis)
```

Comment:
`local_storage` is a constant value and does not need to be recreated for each collection